qa-core/src/tests/internal-api/applications/create.spec.ts

```diff
@@ -185,11 +171,72 @@ describe("Internal API - /applications endpoints", () => {
   })
 
   it("DELETE - Delete an application", async () => {
-    const [response, app] = await config.applications.create(generateApp())
-    expect(response).toHaveStatusCode(200)
-    expect(app.appId).toBeDefined()
+    const app = await config.applications.create(generateApp())
 
     const [deleteResponse] = await config.applications.delete(<string>app.appId)
     expect(deleteResponse).toHaveStatusCode(200)
   })
+
+  it("Operations on Tables", async () => {
```

Comment:
Doesn't seem quote right for this to live under `applications`. Given all the assertions here are specific to a table, this should probably live in the table suite